changelogs/fragments/63467-docker-stack-return-fix.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - docker_stack - Added stdout, stderr, and rc to return values.
```

You can use [ReST](https://en.wikipedia.org/wiki/ReStructuredText) formatting here:
```suggestion
  - docker_stack - Added ``stdout``, ``stderr``, and ``rc`` to return values.
```